src/main/java/frc/robot/Constants.java

```diff
@@ -214,6 +220,94 @@ public static class TELEOP_AUTO_ALIGN {
     }
   }
 
+  public static class constIntake {
+
+    public static final AngularVelocity ZEROED_VELOCITY = Units.RotationsPerSecond.of(0.2);
+
+    public static final Angle MAX_POS = Units.Degrees.of(57);
+    public static final Angle MIN_POS = Units.Degrees.of(-37);
```

Comment:
For elevators we use the term POS or position and for pivots we use the term ANGLE
```suggestion
    public static final Angle MAX_ANGLE = Units.Degrees.of(57);
    public static final Angle MIN_ANGLE = Units.Degrees.of(-37);
```